src/umdsst/Fields/Fields.cc

```diff
@@ -195,12 +196,17 @@ namespace umdsst {
       const double missing_nc = -32768.0;
       for (int j = 0; j < lat; j++)
         for (int i = 0; i < lon; i++)
-          if (abs(sstData[j][i]-(missing_nc)) < epsilon)
+          if (abs(sstData[j][i]-(missing_nc)) < epsilon) {
             sstData[j][i] = missing_;
-          else
+          } else {
             // Kelvin to Celsius which JEDI use internally, will check if the
             // units is Kelvin or Celsius in the future
-            sstData[j][i] -= 273.15;
+            bool isKelvin = false;
+            if (conf.get("kelvin", isKelvin) && isKelvin)
```

Comment:
only comment would be to move `conf.get()` outside of the nested for loops and instead put 
```
bool isKelvin = conf.getBool("kelvin", false);
```

 at the top of the function. (which i think is our preferred syntax of getting an optional parameter with a default value defined). It's fine for now, but if we go to a much higher resolution in the future, repeatedly reparsing the yaml file here at each gridpoint might be slow.

Reply:
Done, good suggestion! Passed all tests.